packages/contentstack-variants/src/export/attributes.ts

```diff
@@ -3,17 +3,18 @@ import { resolve as pResolve } from 'node:path';
 
 import { formatError, fsUtil, PersonalizationAdapter, log } from '../utils';
 import { PersonalizationConfig, ExportConfig, AttributesConfig, AttributeStruct } from '../types';
+import { Command } from '@contentstack/cli-command';
 
 export default class ExportAttributes extends PersonalizationAdapter<ExportConfig> {
   private attributesConfig: AttributesConfig;
   private attributesFolderPath: string;
   private attributes: Record<string, unknown>[];
   public personalizationConfig: PersonalizationConfig;
 
-  constructor(readonly exportConfig: ExportConfig) {
+  constructor(readonly exportConfig: ExportConfig, readonly command: Command) {
     super({
       config: exportConfig,
-      baseURL: exportConfig.modules.personalization.baseURL[exportConfig.region.name],
+      baseURL: exportConfig.modules.personalization.baseURL[exportConfig.region.name] || command.personalizeUrl,
```

Comment:
@harshithad0703 ,Here, you can directly set something similar to the developer hub url.https://github.com/contentstack/cli/blob/development/packages/contentstack-export/src/commands/cm/stacks/export.ts#L108